src/dotnet/UI.Blazor/Components/Navbar/NavbarGroupToggle.razor

```diff
@@ -6,26 +6,31 @@
     var isActive = NavbarUI.ActiveGroupId == Id;
     var cls = isActive ? "on" : "";
     var btnClass = $"navbar-group-toggle {cls} {Class}";
+    var lineClass = isActive ? "bg-success" : "bg-transparent";
 }
 
-<ButtonSquare Click="@OnClicked" Title="@Title" Class="@btnClass">
-    <svg xmlns="http://www.w3.org/2000/svg" class="navbar-group-icon fill-current h-@IconHeight w-@IconWidth" viewBox="@ViewBox">
-        <path d="@IconPath"></path>
-    </svg>
-</ButtonSquare>
+<div class="flex-x">
+    <div class="h-8 w-1 mr-2 self-center rounded-lg @lineClass">
+    </div>
+    <ButtonSquare Click="@OnClicked" Title="@Title" Class="@btnClass">
+        <svg xmlns="http://www.w3.org/2000/svg" class="navbar-group-icon fill-current h-@IconHeight w-@IconWidth" viewBox="@ViewBox">
```

Comment:
NOTE: Tailwind will not include generated classs with expression: h-@IconHeight w-@IconWidth
Do we need arbitrary sizes here?

Reply:
See `SquareSize` and `SquareSizeExt` types - they do it in a Tailwind-friendly way (+ you can add more sizes there if you want to).

Reply:
Curious how @crui3er even noticed this :)